web/thesauruses/java/11/exception_handling.json

```diff
@@ -0,0 +1,316 @@
+            "name": "Are specific exception types an object?",
+            "code": "Yes"
+        },
+        "are_exception_types_gotos": {
+            "name": "Are specific exception types a Goto statement?",
+            "code": "No"
+        },
+        "base_exception_type": {
+            "name": "Base exception type name",
+            "code": "Exception"
+        },
+        "define_new_exception_type": {
+            "name": "Statement to define a new exception type",
+            "code": "class CustomException extends Exception {\n    statements\n}"
+        },
+        "try_or_operator": {
+            "name": "Try-Or operator",
+            "not-implemented": true
+        },
+        "statement_to_start_trying_code": {
+            "name": "Statement to start a block of code to try",
+            "code": "try {\n    statements\n}"
+        },
+        "statement_to_start_catching_exceptions": {
+            "name": "Statement to start catching exceptions",
+            "code": "catch(CustomException e) {\n    statements\n}",
+            "comment": "Here, `CustomException` is the new exception that you have defined"
+        },
+        "statement_to_catch_generic_exception": {
+            "name": "Statement to catch a generic exception",
+            "code": "catch(Exception e) {\n    statements\n}"
+        },
+        "statement_to_catch_specific_exception": {
+            "name": "Statement to catch a specific exception",
+            "code": "catch(CustomException e) {\n    statements\n}",
+            "comment": "Here, `CustomException` is the new exception that you have defined"
+        },
+        "statement_to_catch_multiple_exceptions": {
+            "name": "Statement to catch multiple types of exceptions",
+            "code": "catch(ExceptionType1 | ExceptionType2 | ExceptionType3 e) {\n    statements\n}"
+        },
+        "statement_to_break_out_if_caught_exception": {
+            "name": "Statement to break out of a block if an exception is caught",
+            "not-implemented": true
+        },
+        "statement_to_stop_program_if_caught_exception": {
+            "name": "Statement to stop the program if an exception is caught",
+            "code": "catch(CustomException e) {\n    System.exit(1)\n}",
+            "comment": "The exit command placed in the catch block will terminate the program if an exception of type `CustomException` occurs"
+        },
+        "statement_to_start_code_if_no_exceptions_caught": {
+            "name": "Statement to start code if no exceptions are caught",
+            "code": [
+                "bool success = true;",
+                "try {",
+                "    something();",
+                "} catch (Exception e) {",
+                "    success = false;",
+                "}",
+                "if (success) {",
+                "}"
+            ],
+            "comment": "Not natively possible, but we can use this hack to simulate Python's try-except-else"
```

Comment:
One of the "best practices" is that Code Thesaurus is a language reference tool and not an algorithm reference tool. So if you have to invent the functionality, it shouldn't be in CT. So this should just be a "not-implemented" line instead.

Reply:
(Same for similar places below you implement the functionality too)

Reply:
Makes sense, I've removed this part for all 3 versions. Haven't found any other such instances.